data/json/monsters/nether.json

```
@@ -249,7 +249,7 @@
     "dodge": 3,
     "harvest": "exempt",
     "special_attacks": [ [ "DOGTHING", 40 ] ],
-    "death_function": [ "THING" ],
+    "death_function": [ [ "SPAWN_MON", "mon_thing" ] ],
```

```suggestion
    "death_function": [ [ "SPAWN_MON", "mon_headless_dog_thing" ] ],
```
Not sure if it's intended that killing the dog before it uses its special attack skips the headless dog monster phase? If that was unintended that have that, if intended feel free to mark as resolved.

---

original behavior using `"THING"` directly spawns a `mon_thing` skipping over `mon_headless_dog_thing` so afaik it's intended behavior to do so. This is a horizontal change to the new syntax with no difference in functionality. Internally it will call a function identical to `mdeath::thing`